app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import { Clock, MenuVertical } from "@bigbinary/neeto-icons";
+import * as neetoui from "@bigbinary/neetoui";
```

Comment:
This is not a good practice. Why import everything when you only need a bunch of modules? i believe neetoUI is tree-shakeable.

Refer:
- https://webpack.js.org/guides/tree-shaking/
- https://stackoverflow.com/a/68991103